docs/modules/operation/pages/deep-dive/kafka-producer/configure-kafka.adoc

```diff
@@ -124,20 +124,34 @@ Consult the source code of the `org.opennms.netmgt.model.OnmsAlarm` class in you
 
 To enable metric forwarding, set the value of the `forward.metrics` property to `true`.
 
+[source, console]
+----
+$ ssh -p 8101 admin@localhost
+...
+admin@opennms()> config:edit org.opennms.features.kafka.producer
+admin@opennms()> config:property-set forward.metrics true
+admin@opennms()> config:update
+----
+
 Kafka Producer sends a full CollectionSet as one message to Kafka.
 Large network equipment might have hundreds of thousands of metrics that could generate messages over the Kafka limit (1MB in size by default).
 If the message size is over 1MB , they will broken up into smaller messages that are less than 1MB.
 When sending one CollectionSet over multiple messages, their timestamps will remain same.
+=== Disable metrics splitting into multiple kafka messages
```

Comment:
I think there needs to be a line break right after this so that Antora doesn't treat this as a paragraph. When I look at it in the previews, it's rendered incorrectly.